src/olympia/blocklist/mlbf.py

```diff
@@ -239,11 +239,12 @@ def should_reset_base_filter(self, previous_base_mlbf):
             return True
 
     def blocks_changed_since_previous(self, previous_base_mlbf):
+        self.fetch_blocked_json()
         try:
             # compare base with current blocks
             extras, deletes = self.generate_diffs(
-                previous_base_mlbf.blocked_json, self.fetch_blocked_json())
-            return bool(extras) or bool(deletes)
+                previous_base_mlbf.blocked_json, self.blocked_json)
+            return len(extras) + len(deletes)
         except FileNotFoundError:
             # when previous_base_mlfb._blocked_path doesn't exist
-            return True
+            return len(self.blocked_json)
```

Comment:
I want the function to return the numbers rather than just `True` or `False`.  The only scenario where it would be Falsey rather than Truthy would be when there is no previous base (there was an exception thrown, i.e. it's the first blocklist generated) _and_ there are 0 blocked guids.  We don't particularly support this case on AMO - it's unlikely you'd enable the cron when there are no blocked addons - but returning 0 (Falsey) is probably better anyway, because it wouldn't try to generate an empty bloomfilter.